modules/auxiliary/admin/edirectory/edirectory_dhost_cookie.rb

```diff
@@ -22,7 +22,6 @@ def initialize(info = {})
       },
       'References'     =>
```

Comment:
As there are many modules without references, i would simply remove the references hash when there are no references left.